system-test/bigtable.js

```diff
@@ -372,6 +372,40 @@ describe('Bigtable', function() {
         done
       );
     });
+
+    it('should generate consistency token', done => {
+      TABLE.generateConsistencyToken(function(err, token) {
+        assert.notStrictEqual(token, undefined);
```

Comment:
We don't need this line, since the next confirms it's a string.

Instead, can you add:

```js
assert.ifError(err);
```